Runtime/ApiRequests.cs

```diff
@@ -1,24 +1,30 @@
-﻿using System.Collections.Generic;
+﻿using Newtonsoft.Json;
+using System.Collections.Generic;
 using System.Linq;
 using System.Text;
 using System.Threading.Tasks;
-using Newtonsoft.Json;
 using UnityEngine;
 using UnityEngine.Networking;
 
 namespace BlockadeLabsSDK
 {
-    internal class ApiRequests
+    internal static class ApiRequests
     {
         private static readonly string ApiEndpoint = "https://backend.blockadelabs.com/api/v1/";
 
-        private static async Task<T> GetAsync<T>(string path, string apiKey, params (string, string)[] queryParams)
+        private static async Task<T> GetAsync<T>(string path, string apiKey, Dictionary<string, string> queryParams = null)
         {
-            var queryString = "?api_key=" + UnityWebRequest.EscapeURL(apiKey);
-            if (queryParams.Length > 0)
+            if (!string.IsNullOrWhiteSpace(apiKey))
             {
-                queryString += "&" + string.Join("&", queryParams.Select(kv =>
-                    UnityWebRequest.EscapeURL(kv.Item1) + "=" + UnityWebRequest.EscapeURL(kv.Item2)));
+                queryParams ??= new();
```

Comment:
older version of Unity don't support new()

Reply:
what is the min supported version?